src/components/table/LightTableHead.jsx

```diff
@@ -49,19 +50,23 @@ export class LightTableHead extends React.PureComponent {
                 padding={col.disablePadding ? 'none' : 'default'}
                 sortDirection={orderBy === col.id ? orderDirection : false}
               >
-                <Tooltip
-                  title="Sort"
-                  placement={col.numeric ? 'bottom-end' : 'bottom-start'}
-                  enterDelay={300}
-                >
-                  <TableSortLabel
-                    active={orderBy === col.property}
-                    direction={orderDirection}
-                    onClick={this.createSortHandler(col.property)}
+                {summary ? (
```

Comment:
This is where the tables allow sorting on the columns. For summary tables (like in the Home view), the columns are not clickable buttons.